packages/opentelemetry/eslint.config.js

```diff
@@ -0,0 +1,8 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+
+module.exports = [
```

Comment:
I believe this config should be similar to other non-ui package configs (e.g. the `testing` one) and extend the base config. I see it wasn't doing that, but it should.

Reply:
done